### PR TITLE
[IOTDB-2948] ConfigNode supports ClientPool

### PR DESCRIPTION
**Pull request by CRZbulabula**

### CODE REVIEW

**Comment by CRZbulabula** [comment]
WIP.

File: node-commons/src/main/java/org/apache/iotdb/commons/client/async/AsyncDataNodeInternalServiceClient.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public void close() {
    * return self if clientManager is not null, the method doesn't need to call by user, it will
    * trigger once client transport complete.
    */
-  private void returnSelf() {
+  public void returnSelf() {
```
Comment (OneSizeFitsQuorum) [inline node-commons/src/main/java/org/apache/iotdb/commons/client/async/AsyncDataNodeInternalServiceClient.java:81]: Why change this?

**Review by OneSizeFitsQuorum — COMMENTED** [review]
PTAL

File: node-commons/src/main/java/org/apache/iotdb/commons/client/async/AsyncDataNodeInternalServiceClient.java (hunk 1 above)
Comment (CRZbulabula) [reply node-commons/src/main/java/org/apache/iotdb/commons/client/async/AsyncDataNodeInternalServiceClient.java:81]: Sorry, I forgot to change it back~

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by xingtanzjr — APPROVED** [review]

**Pull request merged**